examples/device/net_lwip_webserver/src/tusb_config.h

```diff
@@ -39,7 +39,7 @@
   #error CFG_TUSB_MCU must be defined
 #endif
 
-#if CFG_TUSB_MCU == OPT_MCU_LPC43XX || CFG_TUSB_MCU == OPT_MCU_LPC18XX || CFG_TUSB_MCU == OPT_MCU_MIMXRT10XX
+#if CFG_TUSB_MCU == OPT_MCU_LPC43XX || CFG_TUSB_MCU == OPT_MCU_LPC18XX || CFG_TUSB_MCU == OPT_MCU_MIMXRT10XX || CFG_TUSB_MCU == OPT_MCU_NUC505
```

Comment:
the highspeed opt is indeed not a ideal solution, though some vendor has dual ports like lpc18xx, which only has a builtin HS PHY for 1 port, the other port is only has FS PHY. I will try a better way to describe the port number/speed selection later on.